src/test/java/io/reactivex/internal/operators/flowable/FlowableRefCountTest.java

```diff
@@ -575,4 +575,21 @@ public void accept(String t1) {
         System.out.println(intervalSubscribed.get());
         assertEquals(6, intervalSubscribed.get());
     }
+
+    private enum CancelledSubscriber implements Subscriber<Integer> {
```

Comment:
`CancellingSubscriber`? Like `DisposingObserver`. It's not cancelled, but it's behavior-subscruber that cancels subscriptions.

My English is not that good so maybe it's false positive from me.